### DISCUSSION TITLE
Custom positions in Quick Throw?

### DISCUSSION THREAD

**Discussion by dtremit** [rxhanson/RectanglePro-Community#445] [Q&A]
Apologies if I've missed this elsewhere, but is it possible to use custom positions in quick throw?

I work on a large 4K display and like to tile windows so they take up either the top 1/3 or bottom 2/3 of the screen — so e.g. a 1920x720 window at top left and a 1920x1440 window at bottom left. 

I've defined those as a custom size and position, and can figure out how to use them in Window Throw, but can't figure out how to use them in Quick Throw. Is there an option I'm missing, or is that not currently possible?

**Comment by rxhanson** [comment]
That's not currently possible, but I'll put your +1 for that feature on my backlog. Thanks for the feedback.

**Reply by tbirdtbird555** [reply]
Would love this feature as well. I use a 49" display for productivity and Rectangle is changing my life right now. Thanks for everything!

**Comment by schutztj** [comment]
+1 - Just got a 49" WQHD display and have segmented it into 4 distinct segments via custom positions and would love to be able to use these in the quick throw configuration.

Just food for thought, but on a monitor this size I actually use it almost like 3 independent surfaces from a windowing perspective. Would be cool if that could be defined such that all the normal shortcuts worked as if each surface was it's own monitor based on where the mouse was e.g. snapping to the left would snap to the left of my middle surface and not all the way to the left of the monitor.

**Reply by rxhanson** [reply]
Thanks for the feedback. I like the splitting display idea. I have some similar ideas but had not thought about it specifically like this. Might be sort of complex to communicate in settings, though?
